gi_loadouts/data/weap/polearms/bsrs.py

```diff
@@ -0,0 +1,19 @@
+from ....type.rare import Rare
+from ....type.weap import Polearm, WeaponStat, WeaponStatType
+from ....type.weap.tier import Tier
+
+
+class BloodsoakedRuins(Polearm):
+    name: str = "Bloodsoaked Ruins"
+    seco_stat: WeaponStat = WeaponStat(stat_name=WeaponStatType.critical_rate_perc, stat_data=4.8)
+    tier: Tier = Tier.Tier_2
```

Comment:
Weapon tier seems incorrect. Thanks to correct it.

```suggestion
    tier: Tier = Tier.Tier_3
```